lib/board.rb

```diff
@@ -31,12 +29,14 @@ def in_progress?
     outcome.status == :in_progress
   end
 
-  private
-
   def next_token
     grid.count(tokens[:X]) > grid.count(tokens[:O]) ? tokens[:O] : tokens[:X]
   end
 
+  private
+
+  attr_reader :grid, :tokens
+
```

Comment:
❤️

Reply:
Too many but that could be the nature of the board as the focus of the game